app/assets/javascripts/controllers/angular-bootstrap/DatepickerCtrl.js

```diff
@@ -0,0 +1,22 @@
+miqAngularApplication.controller('DatepickerCtrl', function ($scope) {
+  $scope.minDate = new Date();
+
+  $scope.disabled = function(date, viewDate, mode) {
+    disabledDate = new Date($scope.$parent.angularForm[viewDate].$viewValue);
+
+    if($scope.$parent.angularForm[viewDate].$untouched) {
+      return ( mode === 'day' && (disabledDate.getUTCDate() === date.getDate()) &&
+                              disabledDate.getUTCMonth() === date.getMonth() &&
+                              disabledDate.getUTCFullYear() === date.getFullYear())
```

Comment:
This feels weird, comparing UTC date with local date.. but it doesn't seem to get used anywhere anyway..